src/Parameters.hs

```diff
@@ -171,6 +181,25 @@ parseDBConnectionString =
         <> help "\"host=localhost port=5432 user=<user> password=<pass>\""
     )
 
+parseLogLevel :: Parser LogLevel
+parseLogLevel =
+  option
+    (maybeReader validateLevel)
+    ( long "log-level"
+        <> metavar "LOG_LEVEL"
+        <> value LevelWarn
```

Comment:
I'm actually not sure about it, maybe `LevelInfo` should be default? It all depends how they are used throughout the codebase

Reply:
I will leave it as `Info` then, the main reason to use `Warn` was to reduce the output in `CTL` logs without the need to provide the `--log-level` parameter.